gordon_gcp/schemas/event.schema.json

```diff
@@ -1,3 +1,49 @@
 {
-    "foo": "bar"
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "id": "https://github.com/spotify/gordon-gcp/blob/master/gordon_gcp/schemas/event.schema.json",
+    "title": "Generic Event Messages",
+    "description": "Schema for parsing generic event messages",
+    "required": ["name", "action", "type"],
+    "properties": {
+        "name": {
+            "description": "Record name",
+            "type": "string",
+            "examples": [
+                "www.example.com.",
+                "some-fqdn.subdomain.example.com."
+            ]
+        },
+        "rrdatas": {
+            "description": "Resource record(s) that `name` will point to",
+            "type": "array",
+            "minItems": 1,
+            "items": {
+                "type": "string"
+            },
+            "uniqueItems": true,
+            "examples": [
+                "198.1.1.50",
+                "example.com.",
+                "some text value"
+            ]
+        },
+        "ttl": {
+            "description": "TTL in seconds; default 300 seconds",
+            "type": "number",
+            "maximum": 86400,
+            "minimum": 300,
+            "default": 300
+        },
+        "type": {
+            "description": "Type of resource record(s) as supported by Google Cloud DNS",
+            "type": "string",
+            "enum": ["A", "AAAA", "CNAME", "MX", "NS", "PTR", "SOA", "SRV", "TXT"]
+        },
+        "action": {
+            "description": "Add or delete resource record sets (rrdatas) listed",
+            "type": "string",
+            "enum": ["additions", "deletions"]
```

Comment:
Mimicking the [changes](https://cloud.google.com/dns/api/v1/changes#resource) endpoint.